apps/console/src/features/users/components/wizard/add-user-wizard.tsx

```diff
@@ -744,50 +1025,314 @@ export const AddUserWizard: FunctionComponent<AddUserWizardPropsInterface> = (
+                            message: t(
+                                "console:manage.features.invite.notifications.sendInvite.error.message"
+                            )
+                        }));
+                    } else {
+                        closeWizard();
+                        // Generic error message
+                        dispatch(addAlert({
+                            description: t(
+                                "console:manage.features.invite.notifications.sendInvite.genericError.description"
+                            ),
+                            level: AlertLevels.ERROR,
+                            message: t(
+                                "console:manage.features.invite.notifications.sendInvite.genericError.message"
+                            )
+                        }));
+                    }
+                })
+                .finally(() => {
+                    setIsSubmitting(false);
+                });
+        } 
+    };
+
+    /**
+     * Basic Wizard Step.
+     * @returns Basic details wizard step.
+     */
+    const getInviteParentOrgUserStep = (): WizardStepInterface => {
+        return {
+            content: (
+                <InviteParentOrgUser
+                    triggerSubmit={ submitGeneralSettings }
+                    onSubmit={ (values: InternalAdminFormDataInterface | UserInviteInterface) => 
+                        sendParentOrgInvitation(values as UserInviteInterface)
+                    }
+                    setFinishButtonDisabled={ (setFinishButtonDisabled) }
+                />
+            ),
+            icon: null,
+            name: WizardStepsFormTypes.BASIC_DETAILS,
+            title: t("console:manage.features.user.modals.addUserWizard.steps.basicDetails")
+        };
+    };
+
+    /**
+     * Resolves the step content.
+     *
+     * @returns Step content.
+     */
+    const resolveStepContent = (): ReactElement => {
+        switch (wizardSteps[ currentWizardStep ]?.name) {
+            case WizardStepsFormTypes.USER_TYPE:
+                return getUserSelectionWizardStep()?.content;
+            case WizardStepsFormTypes.BASIC_DETAILS:
+                return resolveBasicDetailsStep()?.content;
+            case WizardStepsFormTypes.GROUP_LIST:
+                return getUserGroupsWizardStep()?.content;
+            case WizardStepsFormTypes.SUMMARY:
+                return getSummaryWizardStep()?.content;
+            case WizardStepsFormTypes.USER_SUMMARY:
+                return getUserSummaryWizardStep()?.content;
+        }
+    };
+
     const STEPS: WizardStepInterface[] = isAdminUser 
         ? [ ALL_STEPS[0], ...ALL_STEPS.slice(3) ]
         : OrganizationUtils.isCurrentOrganizationRoot()
             ? [ ...ALL_STEPS ]
             : [ ...ALL_STEPS.slice(0, 2), ...ALL_STEPS.slice(3) ];
 
+    const showInternalUserWizard = (): ReactElement => {
+        return (
+            <>
```

Comment:
No need of fragment for elements with one child

Reply:
There are two child components present here.

Reply:
Sorry... didn't notice it.